Latest version of markdown breaks chalice deployments in github workflows

Sample Repo with workflow to reproduce the issue:
https://github.com/apfromiit/markdown-github-workflow

Workflow fails with `markdown==3.3.5`
Workflow runs successfully with `markdown==3.3.4`

## Latest version of markdown breaks chalice deployment in github actions

Workflow: https://github.com/apfromiit/markdown-github-workflow/blob/master/.github/workflows/chalice-deploy.yml

Result: https://github.com/apfromiit/markdown-github-workflow/runs/4229057005?check_suite_focus=true

Refer the workflow in `.github/workflows/` 

### Steps to reproduce:

* Fork the repo 
* Add AWS secrets to forked repo
* Modify Readme (to make some trivial changes to create dummy commit)
* git commit and push to master branch
* Check Github Actions in repo.

Github workflow fails with below logs during `chalice deploy` step.

### Logs:

```
Traceback (most recent call last):
  File "/home/runner/.local/lib/python3.8/site-packages/chalice/cli/__init__.py", line 636, in main
    return cli(obj={})
  File "/usr/lib/python3/dist-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3/dist-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3/dist-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/runner/.local/lib/python3.8/site-packages/chalice/cli/__init__.py", line 189, in deploy
    deployed_values = d.deploy(config, chalice_stage_name=stage)
  File "/home/runner/.local/lib/python3.8/site-packages/chalice/deploy/deployer.py", line 376, in deploy
    return self._deploy(config, chalice_stage_name)
  File "/home/runner/.local/lib/python3.8/site-packages/chalice/deploy/deployer.py", line 382, in _deploy
    self._validate_config(config)
  File "/home/runner/.local/lib/python3.8/site-packages/chalice/deploy/deployer.py", line 408, in _validate_config
    validate_configuration(config)
  File "/home/runner/.local/lib/python3.8/site-packages/chalice/deploy/validate.py", line 41, in validate_configuration
    routes = config.chalice_app.routes
  File "/home/runner/.local/lib/python3.8/site-packages/chalice/config.py", line 141, in chalice_app
    app = v()
  File "/home/runner/.local/lib/python3.8/site-packages/chalice/cli/factory.py", line 277, in load_chalice_app
    app = importlib.import_module('app')
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/runner/work/markdown-github-workflow/markdown-github-workflow/app.py", line 2, in <module>
    from markdown import markdown
  File "/home/runner/.local/lib/python3.8/site-packages/markdown/__init__.py", line 29, in <module>
    from .core import Markdown, markdown, markdownFromFile  # noqa: E402
  File "/home/runner/.local/lib/python3.8/site-packages/markdown/core.py", line 26, in <module>
    from . import util
  File "/home/runner/.local/lib/python3.8/site-packages/markdown/util.py", line 88, in <module>
    INSTALLED_EXTENSIONS = metadata.entry_points(group='markdown.extensions')
TypeError: entry_points() got an unexpected keyword argument 'group'

Error: Process completed with exit code 2.
```

--- a/setup.py
+++ b/setup.py
@@ -74,7 +74,7 @@ setup(
     license='BSD License',
     packages=['markdown', 'markdown.extensions'],
     python_requires='>=3.6',
-    install_requires=["importlib-metadata>='4.4';python_version<'3.10'"],
+    install_requires=["importlib-metadata>=4.4;python_version<'3.10'"],
     extras_require={
         'testing': [
             'coverage',

